apps/token/src/components/token-input/token-input.tsx

```diff
@@ -0,0 +1,192 @@
+import './token-input.scss';
```

Comment:
```suggestion
// TODO use version of this from UI toolkit
import './token-input.scss';
```

Reply:
I already updated this to use Input from ui-toolkit. However Input doesn't support rendering a tag 'inside' the element. So I've created this issue to capture https://github.com/vegaprotocol/frontend-monorepo/issues/273

For the time being we wont render the currentcy inside the input

Reply:
Already using Input from ui-toolkit, but without currency tag